test/models/user_test.rb

```diff
@@ -18,4 +18,11 @@ class UserTest < ActiveSupport::TestCase
 
     refute user.valid?
   end
+
+  test 'a user has an email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.